### ISSUE TITLE
PERSISTENCE

### ISSUE THREAD

**Issue by Megachip** [ikalchev/HAP-python#158]
As requested in the HA repository on issue [#14330](https://github.com/home-assistant/home-assistant/issues/14330) it would be awesome if HAP also support persistence, which is used e.g. on the eve app. 

Thanks a lot.

**Comment by ikalchev** [comment]
Hi,

I would certainly consider this, but I am working on the Camera Accessory right now, so it may take some time. Will keep posted.  Any help is appreciated!

**Comment by djashjones** [comment]
I'd like too see this too

**Comment by oetelaar24** [comment]
Yes, this would be awesome! @ikalchev could you specify what kind of help you would like? Maybe I or someone else can be of some assistance.

**Comment by chriskroees** [comment]
Are there any plans to implement this kind of function? It would be a huge step further for this HomeKit implementation.

**Comment by Pythonaire** [comment]
> Are there any plans to implement this kind of function? It would be a huge step further for this HomeKit implementation.

if you like to test, i made the translation of https://github.com/simont77/fakegato-history here: https://github.com/Pythonaire/FakeGatoHistory. Didn't test all different units but it works at this time for eve room and eve energy.
